doc/classes/@GlobalScope.xml

```diff
@@ -1099,14 +1099,14 @@
 		<method name="rid_allocate_id">
 			<return type="int" />
 			<description>
-				Allocates a unique ID which can be used by the implementation to construct a RID. This is used mainly from native extensions to implement servers.
```

Comment:
This does not feel right, can you elaborate on what I missed?

I would replace all of these RID over the place with a direct reference to the **RID**, but that's for later.

It feels so wrong to me I came back to this message just to try to wrap my head around it. To me, RID reads as it is written. And the acronym stands for "Resource Identifier", not "Identifier Resource".

Reply:
It's "ar I dee", not "rid", this isn't about anything else than grammar